src/SideBar/SideBar.js

```diff
@@ -4,8 +4,8 @@ import './SideBar.css'
 const SideBar = (props) => {
     return (
         <nav
-            className={(props.sidebarIsHidden ? 'd-none' : 'd-block position-fixed position-md-relative') + " sideBar d-md-block col-6 col-md-2 col-lg-1 bg-info p-0 vh-100%"}>
-            <InboxButton allEmailSnippets={props.allEmailSnippets}/>
+            className={(props.sidebarIsHidden ? 'd-none' : 'd-block position-fixed position-md-relative') + " sideBarClass d-md-block col-6 col-md-2 col-lg-1 bg-info p-0 vh-100%"}>
```

Comment:
another problem that was resolved in story 1 and has reemerged - `sideBarClass` should be renamed to `sideBar`